bundle.js

```diff
@@ -0,0 +1,714 @@
+const CONTAINER = document.createElement('div');
+const SCREEN = document.createElement('div');
+const KEYBORD = document.createElement('div');
+
+CONTAINER.classList.add('container');
+KEYBORD.classList.add('keybord');
+document.body.appendChild(CONTAINER);
+SCREEN.classList.add('screen');
+const TA = document.createElement("textarea");
+TA.setAttribute('type', 'text');
+TA.setAttribute('id', 'textarea');
+SCREEN.appendChild(TA);
+CONTAINER.appendChild(SCREEN);
+CONTAINER.appendChild(KEYBORD);
+
+const keyLayout1_rus_down = [
+    'ё', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=',
+    'й', 'ц', 'у', 'к', 'е', 'н', 'г', 'ш', 'щ', 'з', 'х', 'ъ', '\\',
+    'ф', 'ы', 'в', 'а', 'п', 'р', 'о', 'л', 'д', 'ж', 'э',
+    'я', 'ч', 'с', 'м', 'и', 'т', 'ь', 'б', 'ю', '.',
+];
+const keyLayout1_rus_up = [
+    'Ё', '!', '"', '№', ';', '%', ':', '?', '*', '(', ')', '_', '+',
+    'Й', 'Ц', 'У', 'К', 'Е', 'Н', 'Г', 'Ш', 'Щ', 'З', 'Х', 'Ъ', '/',
+    'Ф', 'Ы', 'В', 'А', 'П', 'Р', 'О', 'Л', 'Д', 'Ж', 'Э',
+    'Я', 'Ч', 'С', 'М', 'И', 'Т', 'Ь', 'Б', 'Ю', ',',
+];
+const keyLayout1_eng_down = [
+    '`', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=',
+    'q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p', '[', ']', '\\',
+    'a', 's', 'd', 'f', 'g', 'h', 'j', 'k', 'l', ';', "'",
+    'z', 'x', 'c', 'v', 'b', 'n', 'm', ',', '.', '/',
+];
+const keyLayout1_eng_up = [
+    '~', '!', '@', '#', '$', '%', '^', '&', '*', '(', ')', '_', '+',
+    'Q', 'W', 'E', 'R', 'T', 'Y', 'U', 'I', 'O', 'P', '{', '}', '|',
+    'A', 'S', 'D', 'F', 'G', 'H', 'J', 'K', 'L', ':', '"',
+    'Z', 'X', 'C', 'V', 'B', 'N', 'M', '<', '>', '?',
+];
+const small_rus = [
+    '"', '№', ';', '%',
+    '\\',
+];
+const small_eng = [
+    '@', '#', '$', '^', '&', '|', '"', '<', '>', '?'
+];
+const row_1 = document.createElement('div');
+const row_2 = document.createElement('div');
+const row_3 = document.createElement('div');
+const row_4 = document.createElement('div');
+const row_5 = document.createElement('div');
+row_1.classList.add('row');
```

Comment:
You can create function like this:
function _setRow(node, className) {
    node.classList.add('row');
    KEYBOARD.appendChild(node);
}